k8s/configuration/config-map.yaml

```diff
@@ -0,0 +1,36 @@
+#######################################################
+# This YAML file desribes a ConfigMap for the cluster #
```

Comment:
I wanted a single one with the key value pair.
However, I would have had to change variable names all the other files in case for that to work.
Hence, this was an easy way out.
Also, 2 of them for the Cloud Manager api keys will move to secrets.
I will handle a single config map with changes in the cluster in a separate PR.